### ISSUE TITLE
Training loss at epoch 25 - nan

### ISSUE THREAD

**Issue by CosmicMiriam** [gautham20/pytorch-ts#4]
I try run trainer.train(5, train_dataloader, vd, resume_only_model=True, resume=True) but loss values is  "Training loss at epoch 25 - nan", Do you know if I have to modify the lr? Or is it a data problem?
